rpcs3/Emu/RSX/GL/GLProgramBuffer.h

```diff
@@ -76,11 +76,11 @@ struct GLTraits
 		result->uniforms[1] = GL_STREAM_BUFFER_START + 1;
 
 		rsx_log.notice("*** prog id = %d", result->id());
```

Comment:
This section used to be what "log shader programs" was supposed to do before the DX12 backend came along and we now have an actual shaderlog folder. These lines can be removed or gated behind log shader programs option once more.

Reply:
Solved